target/linux/ath79/base-files/etc/diag.sh

```diff
@@ -17,6 +20,12 @@ set_state() {
 		;;
 	done)
 		status_led_on
+		case "$board" in
+		ligowawe,dlb5-propeller)
```

Comment:
Definitely the wrong place for this code. Other targets are adding custom init scripts to do the same/something similar:

- target/linux/ramips/base-files/etc/init.d/bootcount
- target/linux/mvebu/base-files/etc/init.d/linksys_recovery (same file in multiple targets)

Personally I prefer bootcount to doesn't make it look like something vendor specific.

Reply:
I did it that way for my ar71xx PR that was open for over 10 months.
And I was instructed to do it this was in diag.sh

Reply:
> And I was instructed to do it this was in diag.sh

link?

maybe I miss something. But hijacking the script which handles the diag leds doesn't make any sense to me.

Reply:
@robimarko ar71xx != ath79, please don't copy&paste messy code from old/legacy target. You were instructed to follow/use already existing code instead of duplicating it.